internal/pkg/runtime/engine/singularity/container_linux.go

```diff
@@ -1560,8 +1560,12 @@ func (c *container) getHomePaths() (source string, dest string, err error) {
 	} else {
 		pw, err := user.Current()
```

Comment:
It should also be `user.CurrentOriginal` to get the real user ID because fakeroot when used with unprivileged installation always return root as current user here since this process is running inside fakeroot user namespace